h2/events.py

```diff
@@ -136,6 +136,56 @@ def __repr__(self):
         )
 
 
+class HeadersSent(object):
+    """
+    The HeadersSent event is fired whenever headers are received.  This event
+    carries the HTTP headers and the stream ID of the new stream.
+
+    This is an internal event, used to distinguish between headers
+    and trailers.
+
+    .. versionadded:: 2.5.0
```

Comment:
We don't need a versionadded marker here: that's only necessary for public classes.